fuse_constraints/test/test_marginalize_variables.cpp

```diff
@@ -175,14 +175,15 @@ TEST(MarginalizeVariables, ComputeEliminationOrder)
   auto expected = fuse_constraints::UuidOrdering();
   expected.push_back(x1->uuid());
   expected.push_back(x2->uuid());
-  expected.push_back(x3->uuid());
   expected.push_back(l1->uuid());
+  expected.push_back(x3->uuid());
 
   // Check
   ASSERT_EQ(expected.size(), actual.size());
   for (size_t i = 0; i < expected.size(); ++i)
   {
-    EXPECT_EQ(expected.at(i), actual.at(i));
+    SCOPED_TRACE(i);
```

Comment:
This simply makes the output easier to read and find out which iteration failed.